tensornetwork/quantum/quantum_test.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+import tensornetwork as tn
+import quantum as qu
+
+
+def test_constructor():
```

Comment:
Can you also include `backend` to all of the tests?

Reply:
Ugh, do I have to??

😉 Will do.